Netw4ppl/app/build.gradle

```diff
@@ -12,6 +12,7 @@ sonarqube {
 }
 
 android {
+    namespace 'netw4ppl.ines'
```

Comment:
Can it be changed to SCAN ?

Reply:
Not at the moment = it wil need a BIG refactoring